src/collections/_documentation/workflow/integrations/integration-platform/ui-components.md

```diff
@@ -71,6 +72,26 @@ A `FormField` can be one of three types (which behave like their HTML equivalent
 - uri -(Required if developer doesn't provide `options`) URI to retrieve values from.
 - async - Used only if `uri` is present. If true, will query the URI as the user types, for autocomplete suggestions (see response format below). If false (default), will query the URI once initially to retrieve all possible options. This request *must* succeed, and the response *must* return data in the format Sentry expects, otherwise the entire component won't render.
 - options - (Required if developer doesn't provide `uri`) Static list of options in the format of [label, value]
+- depends_on - (Optional) If a field value depends on the value of another field, a request will be made to load those options when the dependent field is set.
+
+**URI Request Format**
+
+The request sent to the `uri` to load options is a `GET` request. The parameters are encoded into the query parameters as such:
+
+```
+{
+    "installationId": <String>,
+    "projectSlug": <String>,
+    "query": <String>,
+    "dependentData": <String> (optional),
+}
+```
+
+- installationId - The ID of the installation associated with the request.
```

Comment:
```suggestion
- installationId - The ID of the installation associated with the request
```

Reply:
Removing the period to keep it consistent with the rest of Docs and how we handle definitions of parameters.